README.md

```diff
@@ -183,16 +180,23 @@ Examples:
 ```
 
 ### range
-Tests whether a number is in the given range. `range` takes between 1 and 2 number arguments: `min` and `max`.
+Tests whether a number is in the given range. `range` takes 1 or 2 arguments `min` and `max` that can be a number or a value path.
+The range validation can also reference struct values by using `self.` as a value prefix.
 
 Examples:
 
 ```rust
-#[validate(range(min = 1, max = 10))]
+const MAX_CONSTANT: i32 = 10;
+const MIN_CONSTANT: i32 = 0;
+
 #[validate(range(min = 1))]
-#[validate(range(min = 1, max = 10.8))]
+#[validate(range(min = "MIN_CONSTANT"))]
+#[validate(range(min = 1, max = 10))]
 #[validate(range(min = 1.1, max = 10.8))]
 #[validate(range(max = 10.8))]
+#[validate(range(min = "MAX_CONSTANT"))]
+#[validate(range(min = "crate::MAX_CONSTANT"))]
+#[validate(range(min = "self.max"))] // <- References the value max from the struct
```

Comment:
Is that something someone needs? It seems very very niche

Reply:
Yup this is very very specific or better say I have no real use case right now. 

The actual reason why this is here is the `ValueOrPath` enum. I think it would be beneficial to also use it for other things like the `Length` validation as well. I therefor added this functionality which was only four lines:
```rust
if path.starts_with("self.") {
    // Self value
    let ident = syn::Ident::new(path.trim_start_matches("self."), Span::call_site());
    quote!(self.#ident)
}
```

You can decide if we leave it in or not. It's also an easy change otherwise :)

Reply:
Let's remove it for now to keep the API small and easy to understand.

Reply:
It has been removed :)